index.d/minishift.yaml

```diff
@@ -2,7 +2,7 @@ Projects:
   - id: 1
     app-id: minishift
     job-id: dnsmasq
-    git-url: https://github.com/gbraad/dockerfile-dnsmasq
+    git-url: https://github.com/CentOS/container-index.git
```

Comment:
You meant `https://github.com/minishift/dockerfile-dnsmasq-centos`?

Reply:
LOL

Reply:
LOL :)